src/Collapsible.js

```diff
@@ -9,7 +9,7 @@ class Collapsible extends Component {
 
     this.timeout = undefined;
 
-    this.contentId = `collapsible-content-${Date.now()}`;
+    this.contentId = props.contentElementId || `collapsible-content-${Date.now()}`;
```

Comment:
Could we add this prop to the types - and then we could actually use `collapsible-content-${Date.now()}` as the default prop for `contentElementId`

Reply:
Done!

Reply:
Would you mind adding to [index.d.ts](https://github.com/glennflanagan/react-collapsible/blob/develop/index.d.ts) to keep typescript happy too?

Reply:
Done!